infrastructure/application/services/hasura.ts

```diff
@@ -123,7 +126,10 @@ export const createHasuraService = async ({
               name: "HASURA_GRAPHQL_CORS_DOMAIN",
               value: [...CUSTOM_DOMAINS.map((x: any) => x.domain), DOMAIN]
                 .map((x) => `https://*.${x}, https://${x}`)
-                .concat("https://localplanning.services")
+                .concat(
+                  "https://localplanning.services",
+                  "https://planx-website.webflow.io",
```

Comment:
Again, similar to above - what is non-dev webflow URL going to be and should it simply be here already?

Reply:
Good call, will add planx.uk also thanks.